app/backend/src/couchers/servicers/account.py

```diff
@@ -150,3 +157,61 @@ def MarkContributorFormFilled(self, request, context):
             user = session.query(User).filter(User.id == context.user_id).one()
             user.filled_contributor_form = request.filled_contributor_form
         return empty_pb2.Empty()
+
+    def ChangePhone(self, request, context):
+        phone = request.phone
+        # early quick validation
+        if phone and not is_e164_format(phone):
+            context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_PHONE)
+
+        with session_scope() as session:
+            user = session.query(User).filter(User.id == context.user_id).one()
+            if not phone:
+                user.phone = None
+                user.phone_verification_verified = None
+                user.phone_verification_token = None
+                return empty_pb2.Empty()
+
+            if not is_known_operator(phone):
+                context.abort(grpc.StatusCode.UNIMPLEMENTED, errors.UNRECOGNIZED_PHONE_NUMBER)
+
+            if user.phone_verification_sent and now() - user.phone_verification_sent < timedelta(days=180):
```

Comment:
Move 180 to constants too. I think we'll need to tweak these and it'll be easier to have them there